src/compiler/Stage/Parse/Parser.elm

```diff
@@ -410,23 +410,74 @@ literalInt =
         |> P.inContext InLiteralInt
 
 
-{-| TODO escapes
-TODO Unicode escapes
--}
+-- for literalChar and, in the future, literalString
+stringHelp = 
```

Comment:
I'd probably rather name this `character` or `insideQuotes` or something. :thinking:

Reply:
`character` is great because it parses one character, even if that character is represented by multiple bytes (e.g. `\u{123}`) :+1:

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/20/commits/483bf8a270f7a5bb677bb0e09091de2f58655e78